docs/quickstart_inference.md

```diff
@@ -1,52 +1,63 @@
-# 快速推理指引
+# Quick Start Inference
 
-本脚本介绍了102B模型和51B模型快速使用指引，主要包括ckpt转换以及推理服务使用
+This script describes the inference guide of the 102B model and the 51B model, including the ckpt conversion and the use of the inference service.
```

Comment:
This script provides a quick guide on using the 102B model and the 51B model, including instructions for checkpoint (ckpt) conversion and utilizing inference services.